app/models/content_store_payload_version.rb

```diff
@@ -0,0 +1,41 @@
+class ContentStorePayloadVersion < ActiveRecord::Base
+  def self.increment(content_item_id:)
```

Comment:
I don't see how it's different to the `root to: "home#index"` pattern, or `has_many :x, through: :y`